web/src/stream_list.ts

```diff
@@ -846,6 +850,23 @@ export function initialize({
     });
 }
 
+export function initialize_tippy_tooltips(): void {
+    tippy.default("#stream_filters li .subscription_block .stream-name", {
```

Comment:
Why is this using `tippy.default` rather than `tippy.delegate`? The latter is what we use for most tooltips. I'm not sure how those differ.

Reply:
Opened https://chat.zulip.org/#narrow/channel/6-frontend/topic/tippy.20delegate.20vs.20default